KubeArmor/feeder/feeder.go

```diff
@@ -529,10 +532,9 @@ func (fd *Feeder) PushLog(log tp.Log) {
 
 	// standard output / file output
 
-	if fd.filter == "policy" {
-		if len(log.PolicyName) > 0 {
+	if fd.Filter == "policy" {
+		if len(log.PolicyName) > 0 && log.PolicyName != "HostLog" {
```

Comment:
why do we need to filter out HostLog?

Reply:
Its a mistake. I will fix it. I confused Type with PolicyName. I think the conditions based on length are sufficient.